backend/rest/courseRoutes.ts

```diff
@@ -0,0 +1,81 @@
+import { Router } from "express";
+import CourseService from "../services/implementations/courseService";
+import { sendResponseByMimeType } from "../utilities/responseUtil";
+import { CourseUnitDTO } from "../types/courseTypes";
+import { getErrorMessage } from "../utilities/errorUtils";
+import {
+  createCourseUnitDtoValidator,
+  updateCourseUnitDtoValidator,
+} from "../middlewares/validators/courseValidators";
+import { isAuthorizedByRole } from "../middlewares/auth";
+
+const courseRouter: Router = Router();
+const courseService: CourseService = new CourseService();
+
+courseRouter.get(
+  "/",
+  isAuthorizedByRole(new Set(["Administrator"])),
+  async (req, res) => {
+    const contentType = req.headers["content-type"];
```

Comment:
also curious - why the `contentType` for this endpoint in particular (and none of the rest)? is there something different about this one that makes it necessary?

Reply:
i think i just cv pasted how they did this in the starter code. prolly should just do json return instead